src/Pages/TabsPage.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link, useParams } from 'react-router-dom';
+import classnames from 'classnames';
+import { Tab } from '../types/Tab';
+
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+const TabsPage = () => {
+  const { tabId } = useParams();
+  let currentTab = tabs.find(tab => tab.id === tabId) as Tab;
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              key={tab.content}
+              className={classnames({
+                'is-active': currentTab?.id === tab.id,
+              })}
+            >
+              <Link
+                to={`/tabs/${tab.id}`}
+                onClick={() => {
```

Comment:
Remove the onClick handler, as updating the state using useState will automatically trigger the re-rendering.

Reply:
I made the required changes.

Thank you for the input!